llvm/include/llvm/BinaryFormat/ELF.h

```diff
@@ -699,11 +699,19 @@ enum {
 
 // SPARC Specific e_flags
 enum : unsigned {
+  // ELF extension mask.
+  // All values are available for EM_SPARC32PLUS & EM_SPARCV9 objects, except
+  // EF_SPARC_32PLUS which is a EM_SPARC32PLUS-only flag.
+  //
+  // Note that those features are not mutually exclusive (one can set more than
+  // one flag in this group).
```

Comment:
Flags usually are not mutually exclusive, I feel, so I'd be more inclined to call out the case where the flags ARE mutually exclusive.

Reply:
I think it really depends on the context, but I'm not a massive expert specifically on this area of ELF flags, so I can't really advise further. Happy either way.